### ISSUE TITLE
Option to enable/disable AXManualAccessibility, ideally configurable by specific applications

### ISSUE THREAD

**Issue by MichealXie** [dexterleng/vimac#264]
normal condition: 
![2020-11-03 21 11 57](https://user-images.githubusercontent.com/27472703/97989534-a3706580-1e19-11eb-9479-14b1ebd464bd.gif)

after call vimac in vscode (full screen): 
![2020-11-03 21 12 40](https://user-images.githubusercontent.com/27472703/97989642-cb5fc900-1e19-11eb-902b-9cd46e6f46c9.gif)

**Comment by MichealXie** [comment]
I have to restart my system to fix it.

**Comment by dennis-damico** [comment]
also noticing this issue. vscode runs totally fine (scrolls smoothly, for example) but becomes very glitchy the second vimac is run. If you quit vimac and restart vscode it will resolve the issue.

**Comment by dennis-damico** [comment]
while it's possible to mess with vimac to make it ignore vscode i decided it was worth opening an issue in the vscode repo as it seems like this may be a performance issue on their end. The link to that issue can be found [here](https://github.com/microsoft/vscode/issues/110764)

**Comment by dexterleng** [comment]
Vimac enables accessibility on Electron apps with the AXManualAccessibilitt attribute [1].

There's a reason electron apps don't enable accessibility by default unlike native apps. I believe its just that it's not as performant as native.

[1] https://www.electronjs.org/docs/tutorial/accessibility

**Comment by dexterleng** [comment]
Accessibility is enabled when the window is frontmost and disabled when it is no longer frontmost.

I have attempted to only enable it right after entering hint mode (and disable it after exiting hint mode), but there's a race condition problem.

**Comment by dexterleng** [comment]
Until they (Chromium, VSCode) resolve performance issues when using screen readers (not sure how possible), a solution could be a AXManualAccessibility blacklist/whitelist for specific apps. Some apps can take the performance hit to work with Vimac.

**Comment by dennis-damico** [comment]
> blacklist/whitelist for specific apps
I think this is a great solution. Better to use vimac with everything aside from vscode rather than not be able to use it at all. I actually tried implementing that myself but my lack of swift/rxswift made it tough to implement properly. Would be happy to help you with adding that functionality though if possible

**Comment by babramczyk** [comment]
Any update on this? It's the only thing holding me back from using this app extensively 😕
